data.sql

```diff
@@ -14,4 +14,25 @@ INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Pikachu', '01-07-2021', 1, false, 15.04);
 
 INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
-VALUES ('Devimon', '05-12-2017', 5, true, 11);
\ No newline at end of file
+VALUES ('Devimon', '05-12-2017', 5, true, 11);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Charmander', '2020-02-08', 0, false, -11);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Plantmon', '2021-09-15', 2, true, -5.7);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Squirtle','1993-04-02', 3, false, -12.3);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Angemon', '2005-06-12', 1, true, -45);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES('Boarmon', '2005-06-07', 7, true, 20.4);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Blossom', '1998-10-13', 3, true, 17);
```

Comment:
- Hey, the data type for the weight is a floating number(decimal). let's ensure we've passed the correct data type to this column `'17.0'`